packages/hierarchical-tags/webpack.config.js

```diff
@@ -5,6 +5,9 @@ module.exports = {
   output: {
     path: path.resolve(__dirname, "build"),
     filename: "index.js",
+    library: "@benepass/swagger-ui-plugin-hierarchical-tags", // Optional, provide a name for your library
+    libraryTarget: "umd", // Universal Module Definition, to support CommonJS, AMD and browser globals
+    globalObject: "this", // Use 'this' instead of 'window' to support Node.js
```

Comment:
shoutout to chatgpt cus idk how to write a webpack config and the old one was not working.